### PR TITLE
Update Present.md

### PR DESCRIPTION
**Pull request by Kcops11**
Updated my section to more closely align with what we have discussed and agreed to as a team

### CODE REVIEW

File: Present.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-In this section I will talk about where Faker is at currently in terms of its development. Its important to keep in consideration the past of Faker, in all its
-forms. 
+This section will focus on where Faker is currently developing. 
 
-Faker PHP was sunsetted due to design limitations. The creator of PHP faker, noted that the project was relatively large, sitting at 3 mb, and was used for 
-automated testing, which meant it was being downloaded quite frequently. To be specific, the creator estimated that the project was downloaded around 121 million
-times around the time of the sunset announcement. 121 M x 3 = 363 Million MB.
```
Comment (SamRett) [inline Present.md:6]: times around the time of the sunset announcement, meaning 363 million MB was downloaded.

File: Present.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
-In this section I will talk about where Faker is at currently in terms of its development. Its important to keep in consideration the past of Faker, in all its
-forms. 
+This section will focus on where Faker is currently developing. 
 
-Faker PHP was sunsetted due to design limitations. The creator of PHP faker, noted that the project was relatively large, sitting at 3 mb, and was used for 
-automated testing, which meant it was being downloaded quite frequently. To be specific, the creator estimated that the project was downloaded around 121 million
-times around the time of the sunset announcement. 121 M x 3 = 363 Million MB. 
+Let us start by looking at version 6 of Faker, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding the implementation.
```
Comment (SamRett) [inline Present.md:3]: Let us start by looking at version 6 of Faker, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding implementation.

File: Present.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation.
```
Comment (SamRett) [inline Present.md:4]: Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation were added.

**Review by brenno263 — APPROVED** [review]
I think this could use more info on Faker's original design as well. Dig through Marak's early commits a bit.

**Pull request merged**